cobigen/cobigen-core/src/main/java/com/devonfw/cobigen/impl/config/constant/ContextConfigurationVersion.java

```diff
@@ -39,7 +39,15 @@
    * <li>made templateFolder optional</li>
    * </ul>
    */
-  v2_2(2.2f, true);
+  v2_2(2.2f, true),
+
+  /**
+   * ChangeLog:
+   * <ul>
+   * <li>made templateFolder optional</li>
+   * </ul>
+   */
+  v3_0(3.0f, false);
```

Comment:
Is the new version not compatible with the old one? Links and tags are optional, right? So the new version should be compatible with the older versions, shouldnt it?

Reply:
Now technically yes, previously there were changes to the trigger so it wasn't